docker/models/secrets.py

```diff
@@ -4,14 +4,14 @@
 
 class Secret(Model):
     """A secret."""
-    id_attribute = 'ID'
+    id_attribute = 'Id'
 
     def __repr__(self):
-        return "<%s: '%s'>" % (self.__class__.__name__, self.name)
+        return "<%s: '%s'>" % (self.__class__.__name__, self.id)
```

Comment:
Why changing the attribute here?

Reply:
It's because [List Secrets](https://docs.docker.com/engine/api/v1.30/#operation/SecretList) doesn't return the name attribute.

Issue #2025

Reply:
Well... 
The API says it does. That's in the very link you mentioned. If not, it's a bug

Reply:
Name is nested in the `Spec` in the response, so not at the same level as `id`;

```json
[
  {
    "ID": "ktnbjxoalbkvbvedmg1urrz8h",
    "Version": {
      "Index": 11
    },
    "CreatedAt": "2016-11-05T01:20:17.327670065Z",
    "UpdatedAt": "2016-11-05T01:20:17.327670065Z",
    "Spec": {
      "Name": "app-dev.crt"
    }
  }
]
```

<img width="671" alt="Screenshot 2020-02-28 at 17 01 01" src="https://user-images.githubusercontent.com/1804568/75564190-f8b6d480-5a4b-11ea-8263-d9bea2b046fd.png">

Reply:
Hi @ulyssessouza @thaJeztah the [secret creation endpoint](https://docs.docker.com/engine/api/v1.41/#operation/SecretCreate)  just return the `ID` value, so the method `__repr__` raise an exception:

```python
     12     @property
     13     def name(self):
---> 14         return self.attrs['Spec']['Name']
     15 
     16     def remove(self):

KeyError: 'Spec'
```

![image](https://user-images.githubusercontent.com/1179500/109204700-f80f6600-77a5-11eb-919c-6bb0bd31cb44.png)